tvb_nest/interfaces/base.py

```diff
@@ -161,15 +161,15 @@ def nest_state_to_tvb_parameter(self, model):
             # Update TVB parameter
             param_values = getattr(model, interface.name)
             if interface.model == "spike_detector":
-                param_values[self.nest_nodes_ids] = \
+                param_vals = \
```

Comment:
Why this change ?

Reply:
param_values is a numpy ndarray. It is immutable.. so we cannot assign a value to it (it is a read-only attribute). However, if we want, we can force this ndarray to become mutable, by setting the writable flag to True (something like: 'name.setflags(write=True)'). But after we solve this problem, there is another one: param_values looks like this: [ 1. ] … But self.nest_nodes_ids looks like: [34 35]. So we try to set some values in an ndarray at some out of bound locations. I don't understand exactly why this happens (only when running with library 2.0), but I observed (at first sight) that is not used in critical places in project, so I have decided to use for the moment a new variable for this purpose.